sycl/test-e2e/WorkGroupScratchMemory/dynamic_alloc_local_accessor.cpp

```diff
@@ -2,7 +2,7 @@
 // RUN: %{run} %t.out
 //
 
-// UNSUPPORTED: gpu-intel-gen12
+// UNSUPPORTED: gpu-intel-gen12, cpu
```

Comment:
Are these tests supposed to pass on PVC?  They seem to be failing on PVC.  Note that right now we don't have any PVC machines in the CI infra, so it might have snuck by.  Does anyone know whether they should pass or fail on PVC?